packages/react-redux/src/slices/cartoSlice.js

```diff
@@ -190,6 +191,9 @@ export const createCartoSlice = (initialState) => {
       },
       setFeatureSelectionEnabled: (state, action) => {
         state.featureSelectionEnabled = action.payload;
+      },
+      setLocale: (state, action) => {
+        state.locale = action.payload;
```

Comment:
Maybe it's fine to have it in redux, but the problem is that even having it inside, we're not able to get it from any part of the lib (as the UI), and we must pass as prop to widgets. 

This is due the current split in several modules of the lib, maybe we should reconsider this in the future, and things like this would be much easier.

Reply:
As formatters were out of the scope initially, I didn't consider those here. In our current codebase, formatters can be passed down to widgets as props